products/public-websites/Cerner-Support/how-to-move-facility-to-cerner.md

```diff
@@ -0,0 +1,42 @@
+# How to move a non-Cerner facility to being a Cerner facility
```

Comment:
The way this is phrased kinda sounds like we are performing some sort of migration of data into Cerner's systems when we are really just setting up a link to a Cerner health care product for a particular VAMC. But this is really tricky to phrase because there are so many layers...Maybe `How to set up a VAMC's Cerner integration within the VA.gov health care portals`? That is a mouthful

Reply:
That works for me 🙂